contracts/SignerBondsEscrow.sol

```diff
@@ -28,9 +28,19 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 ///         signer bonds so that governance can later swap them manually and
 ///         deposit as coverage pool collateral.
 contract SignerBondsEscrow is ISignerBondsSwapStrategy, Ownable {
-    /// @notice Swaps signer bonds.
+    /// @notice Receive ETH upon withdrawal of risk manager's signer bonds.
+    receive() external payable {}
+
+    /// @notice Swaps the given signer bonds amount from the given risk manager.
     /// @dev Adds incoming bonds to the overall contract balance.
-    function swapSignerBonds() external payable override {}
+    /// @param riskManager Address of the risk manager
+    /// @param amount Amount of signer bonds being swapped.
+    function swapSignerBonds(RiskManagerV1 riskManager, uint256 amount)
```

Comment:
Don't we need `onlyOwner` modifier here? It looks that anyone can call this now.

Reply:
Anyone can invoke the `swapSignerBonds` but only the owner can withdraw.

Reply:
Right, I just wonder if there's a malicious case when one can call this function and not allow to swap bonds on `SignerBondsUniswapV2` contract cause ETH are in this contract. I guess governance can call then `withdraw(SignerBondsUniswapV2.address)` but that's just an additional step for them.

Reply:
`SignerBondsEscrow` has nothing to do with `SignerBondsUniswapV2`. It's a separate strategy.

Reply:
Ah, I overlooked the fact that only 1 strategy can be assigned to a RiskManager at a time.